Orchestrator/docs/FAQ.md

```diff
@@ -0,0 +1,59 @@
+# Frequently Asked Questions
+
+### What are the primary scenarios supported by Orchestrator?
+Primary scenario currently supported is routing/dispatching (intent only).  We are working on supporting entities extraction for future releases.
+
+
+### Can Orchestrator be used from Composer?
+Yes.  Follow documentation [here][13] to use Orchestrator as recognizer in Composer.
+
+### Can you use both Orchestrator and LUIS in the same bot?
+Yes, definitely.  You could use Orchestrator (OrchestratorRecognizer) to route incoming utterances to the right child applications (i.e., LUISRecognizer, QnAMakerRecognizer).
+
+
+### What are the guidelines for when Orchestrator correctly predicts incoming utterance for sub intent (app/skill) but the sub app fails to return the right intent?
+We recommend tuning the sub app models (i.e. LUIS/QnAMaker/Orchestrator) using the recommended best practices for the app.  
+- To improve LUIS model, follow instructions [here][1].
+- To improve QnAMaker kb, follow guidelines [here][2].  
+- To improve Orchestrator, follow guidelines [here][3].
+
+
+### Which languages will be supported in Orchestrator multilingual models?
+The model supports 100 languages (full list can be found at [XLMR Supported Languages][4]). 8 languages (EN, ES, DE, FR, IT, JA, PT, and ZH) are fine-tuned with additional data.  We are constantly looking to expand the number of supported languages and if a language is not yet supported, Azure translation could be used to translate incoming utterance to English first before passing it to Orchestrator.  
```

Comment:
From @hcyang : 
`models were pre-trained from 100+ languages, then further fine-tuned for the 8 languages.`

Reply:
The model supports 100 languages => The model was pre-trained with 100+ languages

Reply:
Took out the 100 languages words and instead added link to NLR Models.